src/main/java/org/zalando/planb/provider/CassandraProperties.java

```diff
@@ -17,6 +19,8 @@
     private int port = ProtocolOptions.DEFAULT_PORT;
     private ConsistencyLevel writeConsistencyLevel = ONE; // TODO what is a sane default?
```

Comment:
That would also be an option but depends a lot on how we want to treat
region issues.

On Friday, 19 February 2016, Henning Jacobs <notifications@github.com>
wrote:

> In src/main/java/org/zalando/planb/provider/CassandraProperties.java
> <https://github.com/zalando/planb-provider/pull/21#discussion_r53506543>:
>
> > @@ -17,6 +19,8 @@
> >      private int port = ProtocolOptions.DEFAULT_PORT;
> >      private ConsistencyLevel writeConsistencyLevel = ONE; // TODO what is a sane default?
>
> Shouldn't we use EACH_QUORUM on write? See zalando/planb-revocation#29
> <https://github.com/zalando/planb-revocation/issues/29>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zalando/planb-provider/pull/21/files#r53506543>.
>


-- 
Jan Mußler
Senior Database Engineer | +49 176 1275 8727 | Zalando SE